concent_api/api-e2e-force-accept-or-reject-results-test.py

```diff
@@ -368,6 +399,8 @@ def test_case_2a_send_duplicated_force_subtask_results(cluster_consts: ProtocolC
 if __name__ == '__main__':
     try:
         from concent_api.settings import CONCENT_PUBLIC_KEY
+        cluster_name = get_cluster_name_from_address(sys.argv[1])
```

Comment:
Same here, I think it should be done inside `SCIBaseTest`.

Reply:
Fixed - moved